misc/review/MidiSendMessageResultsEnum.idl

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+
+namespace MIDI_MIDL_NAMESPACE
+{
+
+    MIDI_API_CONTRACT_v1
+    [flags]
+    enum MidiSendMessageResults
+    {
+        Succeeded =                         0x80000000,
+        Failed =                            0x10000000,
+
+        BufferFull =                        0x00010000,
+        EndpointConnectionClosedOrInvalid = 0x00040000,
+
+        InvalidMessageTypeForWordCount =    0x00100000,
+        InvalidMessageOther =               0x00200000,
+        DataIndexOutOfRange =               0x00400000,
+        TimestampOutOfRange =               0x00800000,
+
+        MessageListPartiallyProcessed =     0x00A00000,
```

Comment:
Clients will be checking the results. Since `MessageListPartiallProcessed` is equal to `TimestampOutOfRange | InvalidMessageOther`, somebody who does

```cpp
if (result & MidiSendMessageResults::MessageListPartiallProcessed) ...
```
will execute even if the value is just TimestampOutOfRange. Is that intended?

Reply:
Nice catch. No, not intended.